server/auth/login.go

```diff
@@ -75,3 +88,69 @@ func randToken() string {
 	rand.Read(b)
 	return base64.StdEncoding.EncodeToString(b)
 }
+
+func SendNickName(message *messageService.Authentification)(string, error){
+	db, err := database.GetStorage()
+	if err != nil {
+		loger.Log.Errorf("Failed to open db andsend contacts", err)
```

Comment:
strange because, I run tests and that logging error is not showing where code is broke, so I set double check